Fix 5.0 community feedback (#1516)
emqtt/emq-docs-cn: en_US/security/authn/authn.md

| JWT             |                   | [Authentication using JWT](./jwt.md)                         |
| SCRAM           | Built-in Database | [Authentication using SCRAM](./scram.md)                     |

## Anonymous login

EMQX 5.0 removes the `alow_anonymous` configuration item. All client connections are allowed by default.

> If **add and enable** any authenticator, EMQX will perform an authentication check for all new connections.

Reviewer: ```suggestion
If authenticator(s) are **added and enabled**, EMQX will perform the authentication check for new connections.
```
